File: src/main/java/com/natint/model/provider/boi/xml/mapping/Currency.java
package com.natint.model.provider.boi.xml.mapping;

public class Currency {
    private String currencycode;
    private String rate;

    public Currency() {
    }

    public Currency(String currencycode, String rate) {
        this.currencycode = currencycode;
        this.rate = rate;
    }

    public String getCurrencycode() {
        return currencycode;
    }

    public void setCurrencycode(String currencycode) {
        this.currencycode = currencycode;
    }

    public String getRate() {
        return rate;
    }

    public void setRate(String rate) {
        this.rate = rate;
    }
}

File: src/main/kotlin/com/natint/springboot/domain/provider/boi/xml/mapping/Currency.kt
package com.natint.springboot.domain.provider.boi.xml.mapping

class Currency {
    private var currencycode: String = ""
    private var rate: String = ""

    constructor()

    constructor(currencycode: String, rate: String) {
        this.currencycode = currencycode
        this.rate = rate
    }

    fun getCurrencycode() = currencycode
    fun getRate() = rate
    fun setCurrencycode(currencycode: String) {
        this.currencycode = currencycode
    }

    fun setRate(rate: String) {
        this.rate = rate
    }
}
